saul-core/src/test/scala/JoinTrainingTests/intializeSparseNetwork.scala

```diff
@@ -0,0 +1,90 @@
+package JoinTrainingTests
```

Comment:
The package name should be `edu.illinois.cs.cogcomp.saul.classifier`, like the the `jointTraining` object: https://github.com/kordjamshidi/saul/blob/38512bc6633c38ad62d75569c255f4571b104a7e/saul-core/src/main/scala/edu/illinois/cs/cogcomp/saul/classifier/JointTrain.scala